### PR TITLE
[BACKUP] Add CRR functionality for Azure Workload

### PR DESCRIPTION
**Pull request by akshayneema**
**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
BACKUP

File: src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_afs_commands.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 
 class BackupTests(ScenarioTest, unittest.TestCase):
-    # @record_only()
+    @record_only()
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_afs_commands.py:26]: Why re-mark these tests as `@record_only()`?

File: src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py [hunk 2]
```diff
@@ -70,82 +70,82 @@ def test_backup_scenario(self, resource_group, vault_name, vm_name, storage_acco
...
-            self.check("length([?resourceGroup == '{rg}'])", number_of_test_vaults - 1),
-            self.check("length([?name == '{vault1}'])", 1),
-            self.check("length([?name == '{vault2}'])", 1),
-            self.check("length([?name == '{vault3}'])", 1)
-        ])
+    #@record_only()
+    #@ResourceGroupPreparer(location="southeastasia")
+    #@VaultPreparer(parameter_name='vault1')
+    #@VaultPreparer(parameter_name='vault2')
+    #def test_backup_vault(self, resource_group, resource_group_location, vault1, vault2):
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py:77]: May I ask why block this test?

File: src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py [hunk 3]
```diff
@@ -496,50 +494,50 @@ def test_backup_restore(self, resource_group, target_resource_group, vault_name,
...
+
+    #    self.cmd('backup job show -g {rg} -v {vault} -n {job3} --use-secondary-region', checks=[
+    #        self.check("properties.entityFriendlyName", vm_name),
+    #        self.check("properties.operation", "CrossRegionRestore"),
+    #        self.check("properties.status", "Completed")
+    #    ])
 
     @ResourceGroupPreparer(location="southeastasia")
-    @VaultPreparer()
+    @VaultPreparer(soft_delete=False)
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py:540]: May I ask why disable the soft deletion~

**Comment by zhoxing-ms** [comment]
@akshayneema Could you please address those conflicts and comments?

**Comment by zhoxing-ms** [comment]
@akshayneema  Please solve these comments and conflicts in time today, otherwise this PR will miss the release of this sprint and can only be released in the next sprint (09-07)

**Comment by zhoxing-ms** [comment]
Since I haven't received any reply for a long time, so I close it first. If we still need merge it in the future, then we can reopen it

**Pull request closed without merging**